linux_build.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+RED='\033[0;31m'
+BG='\033[0;32m'
+NC='\033[0m'
+
+error() {
+    >&2 echo -e "${RED}$1${NC}";
+}
+
+info() {
+    echo -e "${BG}$1${NC}";
+}
+
+mkdir -p build
+cd build
+cmake -DCMAKE_BUILD_TYPE=Debug ..
+
+FLAGS="-j8 -Wall"
```

Comment:
Why does this have to be a variable, when it's only used in a single place?

Travis only offers up to 2 CPUs, setting 8 jobs will only cause thrashing. Why not use ninja instead?

`-Wall` seems like a compiler option, not make one, what is the purpose of this?